_studio/shared/asc/include/asc_common_impl.h

```diff
@@ -178,7 +178,7 @@ static inline __m256i LoadPartialYmm(unsigned char *pSrc, mfxI32 len)
 #pragma warning(disable:4505)
 #endif
 
-static void calc_RACA_4x4_C(mfxU8 *pSrc, mfxI32 pitch, mfxI32 *RS, mfxI32 *CS) {
+static inline void calc_RACA_4x4_C(mfxU8 *pSrc, mfxI32 pitch, mfxI32 *RS, mfxI32 *CS) {
```

Comment:
Fix warning "Wunused-function". This function use 2 times in _studio/shared/asc/src/asc_sse4_impl.cpp and in _studio/shared/asc/src/asc_c_impl.cpp

Reply:
This function use in _studio/shared/asc/src/asc_c_impl.cpp:218. Moreover, all other functions in header file, like LoadPartialYmm, defined as "static inline"

Reply:
In header file this function is not wrapped in any macros